main/esp_app_main.c

```diff
@@ -405,8 +407,8 @@ void register_default_nvs(){
 	ESP_LOGD(TAG,"Registering default value for key %s", "stats");
 	config_set_default(NVS_TYPE_STR, "stats", "n", 0);
 
-	ESP_LOGD(TAG,"Registering default value for key %s", "rel_api");
```

Comment:
Please don't remove these these 2 lines as they are now critical for the OTA process

Reply:
Noticed that too.  Think I missed a commit.  Will update and address Philippes notes in the next few days.